<launch>
  <arg name="out_imu_topic" default="imu_data" description="topic"/>
  <arg name="out_hz" default="1" description="[rad/s]"/>
  <arg name="Threshold_x" default="140" description="[rad/s]"/>
  <arg name="Threshold_y" default="4" description="[rad/s]"/>

  <node pkg="cgi430_can_driver" exec="cgi430_can_driver_node" name="cgi430_can_driver_node" output="screen">
    <param name="out_imu_topic" value="$(var out_imu_topic)"/>
    <param name="out_hz" value="$(var out_hz)"/>
    <param name="Threshold_x" value="$(var Threshold_x)"/>
    <param name="Threshold_y" value="$(var Threshold_y)"/>
  </node>
</launch>
